snpdb/variant_pk_lookup.py

```diff
@@ -75,17 +75,17 @@ def get_loci_ids(self, loci_hashes: Iterable, validate_not_null=True) -> List[st
     def filter_non_reference(self, variant_hashes, variant_ids) -> List:
         pass
 
-    def get_variant_coordinate_hash(self, chrom, position, ref, alt):
+    def get_variant_coordinate_hash(self, chrom, start, end, ref, alt):
         """ For VCF records (needs GenomeBuild supplied) """
         if self.chrom_contig_id_mappings is None:
             raise ValueError("Need to initialise w/GenomeBuild to call get_variant_coordinate_hash")
         contig_id = self.chrom_contig_id_mappings[chrom]
         ref_id = self.sequence_pk_by_seq[ref]
         alt_id = self.sequence_pk_by_seq[alt]
-        return self._get_variant_hash(contig_id, position, ref_id, alt_id)
+        return self._get_variant_hash(contig_id, start, end, ref_id, alt_id)
 
-    def add(self, chrom, position, ref, alt):
-        variant_coordinate = (chrom, position, ref, alt)
+    def add(self, chrom, start, end, ref, alt):
```

Comment:
add method adds a tuple of chrom, start, end, ref, alt to
variant_coordinates (which is type hinted as List[VariantCoordinate]) so it should actually make a VariantCoordinate object